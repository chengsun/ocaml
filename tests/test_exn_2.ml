let f () = ()

let g () = raise Not_found

let _ = try f () with Not_found -> (print_string "wrong: Not_found"; print_newline ())

let _ = try g () with Division_by_zero -> (print_string "wrong: Division_by_zero"; print_newline ())
