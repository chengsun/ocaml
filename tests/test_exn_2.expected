Uncaught OCaml exception: Not_found
